source/elements/oneMKL/source/domains/dft/compute_backward.rst

```diff
@@ -75,203 +87,281 @@ compute_backward (Buffer version)
                              sycl::buffer<input_type, 1>  &in_im,
                              sycl::buffer<output_type, 1> &out_re,
                              sycl::buffer<output_type, 1> &out_im);
-
    }
 
+
 .. container:: section
 
    .. rubric:: Input Parameters
 
    :ref:`desc<onemkl_dft_descriptor>`
-      A fully configured and committed discrete Fourier transform descriptor class object, defining the type of backward transformation and data layout to be applied. At commit time, the ``sycl::queue`` has already been provided.
+      A fully configured and committed object of the
+      :ref:`descriptor<onemkl_dft_descriptor>` class, whose configuration is not
+      inconsistent with backward DFT calculations.
 
    inout
-      Sycl buffer containing an array of length no less than is specified at the :ref:`descriptor construction<onemkl_dft_descriptor_constructor>` time to house both the input and output data sequences for the in-place transformation. Corresponds to the choice of ``config_value::INPLACE`` for the configuration parameter ``config_param::PLACEMENT``.
-
+      ``sycl::buffer`` object of sufficient capacity to store the elements
+      defining all the relevant data sequences, as configured by ``desc``
+      (configured for in-place operations and not with
+      :ref:`onemkl_dft_complex_storage_real_real`, if complex).
 
    inout_re
-      Sycl buffer containing an array of length no less than is specified at the :ref:`descriptor construction<onemkl_dft_descriptor_constructor>` time to house the real part of both the input and output data sequences for the in-place transformation when using the ``config_value::REAL_REAL`` format for the ``config_param::COMPLEX_STORAGE`` configuration parameter. Corresponds to the choice of ``config_value::INPLACE`` for the configuration parameter ``config_param::PLACEMENT``.
+      ``sycl::buffer`` object of sufficient capacity to store the elements
+      defining the real parts of all the relevant data sequences, as configured
+      by ``desc``, complex descriptor configured for in-place operations with
+      :ref:`onemkl_dft_complex_storage_real_real`.
 
    inout_im
-      Sycl buffer containing an array of length no less than is specified at the :ref:`descriptor construction<onemkl_dft_descriptor_constructor>` time to house the imaginary part of both the input and output data sequences for the in-place transformation when using the ``config_value::REAL_REAL`` format for the ``config_param::COMPLEX_STORAGE`` configuration parameter. Corresponds to the choice of ``config_value::INPLACE`` for the configuration parameter ``config_param::PLACEMENT``.
+      ``sycl::buffer`` object of sufficient capacity to store the elements
+      defining the imaginary parts of all the relevant data sequences, as
+      configured by ``desc``, complex descriptor configured for in-place
+      operations with :ref:`onemkl_dft_complex_storage_real_real`.
```

Comment:
```suggestion
      configured by ``desc``. Only with complex descriptor configured for in-place
      operations with :ref:`onemkl_dft_complex_storage_real_real`.
```
And other occurrences.

Reply:
Changed in [be475cc88cd502a9f118c6913dc75b2eae3ad8a2](https://github.com/oneapi-src/oneAPI-spec/pull/506/commits/be475cc88cd502a9f118c6913dc75b2eae3ad8a2)